galata/test/documentation/data/extensions-search.json

```diff
@@ -0,0 +1,190 @@
+{
+  "objects": [
+    {
+      "package": {
+        "name": "jupyterlab-drawio",
+        "scope": "unscoped",
+        "version": "0.9.0",
+        "description": "A JupyterLab extension for embedding drawio / mxgraph.",
+        "keywords": [
+          "extension",
+          "jupyter",
+          "jupyterlab",
+          "jupyterlab-extension"
+        ],
+        "date": "2021-04-29T10:22:28.814Z",
+        "links": {
+          "npm": "https://www.npmjs.com/package/jupyterlab-drawio",
+          "homepage": "https://github.com/QuantStack/jupyterlab-drawio",
+          "repository": "https://github.com/wolfv/jupyterlab-drawio",
+          "bugs": "https://github.com/QuantStack/jupyterlab-drawio/issues"
+        },
+        "author": {
+          "name": "Wolf Vollprecht"
+        },
+        "publisher": {
+          "username": "hbcarlos",
+          "email": "carlosherrerocontact@gmail.com"
```

Comment:
Although it's true they are public on npm, maybe we could use placeholder email addresses here to avoid making them more visible and searchable via the GitHub UI?

Reply:
Good thinking. Done. 👍